### PR TITLE
Fixes #30127 - always skip orchestration

### PR DESCRIPTION
**Pull request by ezr-ondrej**
There is no point in having the method - noone is using it.
Followup on https://github.com/theforeman/foreman/pull/7749#discussion_r464136079

### CODE REVIEW

File: app/services/host_fact_importer.rb [hunk 1]
```diff
@@ -48,7 +44,7 @@ def parse_facts(facts, type, source_proxy)
 
   def skiping_orchestration(&block)
```
Comment (ekohl) [inline app/services/host_fact_importer.rb]: Unrelated, but typo?

**Review by ekohl — COMMENTED** [review]
I always like it when a commit message actually has information. This should say something like:

> bb84e41d391ba1b18dc9e2db8f0cbabf92a3b1e8 removed the setting where this made sense. Without the setting, the method no longer has any use. There are no subclasses that could override it so the method can be dropped.

This gives you some context as a reviewer / looking through git log.

**Comment by ezr-ondrej** [comment]
Love it :+1:
Added the message and fixed typo :)

**Review by ekohl — COMMENTED** [review]
I'd expect line wrapping in the commit message. Also looks like the NPM vendor package has issues and is causing the tests to fail.

**Comment by ezr-ondrej** [comment]
It's not vendor, it's npm registry
EDIT: was npm registry [test foreman]

**Comment by ezr-ondrej** [comment]
[test foreman]

File: app/services/host_fact_importer.rb [hunk 2]
```diff
@@ -31,24 +27,24 @@ def import_facts(facts, source_proxy = nil)
...
     end
   end
 
   private
 
-  def skiping_orchestration(&block)
+  def skipping_orchestration(&block)
     if host.is_a?(Host::Managed)
-      host.without_orchestration_if(skip_orchestration?, &block)
+      host.without_orchestration_if(true, &block)
```
Comment (tbrisker) [inline app/services/host_fact_importer.rb]: That's the default value, so no need to actually pass around `&block`:
```suggestion
      host.without_orchestration_if
        yield
      end
```
Also, considering that this is the only consumer of this method, do we actually need the parameter or should it just be renamed to `.without_orchestration` and the logic of whether or not it needs skipping will stay with the caller?

Comment (ezr-ondrej) [reply app/services/host_fact_importer.rb]: I guess it's fair :+1:

File: app/services/host_fact_importer.rb [hunk 3]
```diff
@@ -31,24 +27,24 @@ def import_facts(facts, source_proxy = nil)
...
     end
   end
 
   private
 
-  def skiping_orchestration(&block)
+  def skipping_orchestration(&block)
     if host.is_a?(Host::Managed)
-      host.without_orchestration_if(skip_orchestration?, &block)
+      host.without_orchestration(&block)
```
Comment (tbrisker) [inline app/services/host_fact_importer.rb]: no need to pass `&block` around, you can also drop it from the method signatures
```suggestion
      host.without_orchestration do
        yield
      end
```

Comment (ezr-ondrej) [reply app/services/host_fact_importer.rb]: It seems to me bit more straight forward :thinking:
It is expecting a `block`, why shouldn't it have `block` in the signature?

Comment (ezr-ondrej) [reply app/services/host_fact_importer.rb]: No strong feelings over it though

Comment (adamruzicka) [reply app/services/host_fact_importer.rb]: Afaik capturing the block as a proc and storing it in a variable is slower than yielding into it directly, but I'm a bit foggy on the details why it is like this

Comment (ezr-ondrej) [reply app/services/host_fact_importer.rb]: hmm didn't find anything on the topic, but I guess I'm ok with changing it :shrug: :)

**Review by tbrisker — APPROVED** [review]
Thanks @ezr-ondrej !

**Comment by lzap** [comment]
I searched all my plugin checkouts and I could not find a reference, good job :-)

**Pull request merged**